### PR TITLE
Don't bail if there is no resend padding packet

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: src/media/inner.rs [hunk 1]
```diff
@@ -579,23 +579,23 @@ impl MediaInner {
...
-                assert!(self.resends.is_empty());
-                self.resends.push_front(Resend {
-                    pt,
-                    ssrc: packet.meta.ssrc,
-                    seq_no,
-                    queued_at: now,
-                });
-
-                return self.poll_packet_resend(now, true);
+                if let Some(packet) = buffer.historic_packet_smaller_than(max_size) {
```
Comment (algesten) [inline src/media/inner.rs:582]: Can you avoid reindentation by using let-else and early return?

**Review by algesten — CHANGES_REQUESTED** [review]

File: src/media/inner.rs (hunk 1 above)
Comment (k0nserv) [reply src/media/inner.rs:582]: No because the other block we want to run is in the same function

Comment (k0nserv) [reply src/media/inner.rs:582]: The problem before was that this code did an early return here, that's exactly what we don't

**Review by algesten — APPROVED** [review]
Alright

**Pull request merged**